### ISSUE TITLE
[Flip to Shhh] Support bringup

### ISSUE THREAD

**Issue by imachabeich** [Magisk-Modules-Repo/Pix3lify#26]
**Describe the bug**
After installing the module, there is no option to enable this feature. 

**Expected behavior**
There should be an option to enable 'Flip to Shhh'.

**Smartphone (please complete the following information):**
 - Device: [Samsung Galaxy S4 and Samsung galaxy S6]
 - OS: [e.g. Android 9.0]
 - Version [2.1.2]

**Comment by TheGabrielHoward** [comment]
It appears for me, but the function isn't enabled

**Comment by ThatNerdyPikachu** [comment]
Yeah, we still need to enable something, as seen [here](https://gist.github.com/thehappydinoa/aa598c3e51c0f67716174b947ddf50f2).

**Comment by thehappydinoa** [comment]
I believe the issue lies within the `mo1654d` function
 
```java
this.f9702f.getPackageManager().getPackageInfo(this.f9715s.mo740a(), 0).getLongVersionCode() < this.f9714r
``` 
evaluates to True and throws 

```java
f9697a.mo1036a(Level.WARNING)).mo1048a("com/google/android/apps/wellbeing/autodnd/impl/AutoDndManagerImpl", "setAutoDndEnabled", 294, "AutoDndManagerImpl.java")).mo1050a("Auto DND is not supported and should not be set as enabled.")
```

**Comment by ThatNerdyPikachu** [comment]
@imachabeich Btw, where are you checking? (You should be checking in ``System > Gestures``)

**Comment by rap4ytm** [comment]
Hi. I Had tryed pix3lify but I can't use the Call screening feature. Is this normal? Have somebody a idea what I can do?

**Comment by imachabeich** [comment]
Yeah, I checked in System > Gestures and System > Digital Wellbeing.

**Comment by lzlrd** [comment]
I have a similar issue too, we're working on it @imachabeich.

**Comment by AuroraWright** [comment]
https://forum.xda-developers.com/pixel-2-xl/themes/enable-flip-to-shhh-pixel-devices-t3907892 try the apk within my module

**Comment by thehappydinoa** [comment]
@AuroraWright I will check out adding this now.

**Comment by lzlrd** [comment]
@AuroraWright it's added in here https://github.com/Magisk-Modules-Repo/Pix3lify/pull/76/commits/eb1d06366b04a167907c0d96ab944d561c06a521.

EDIT: After some testing by @Stallix your port still doesn't work. There may be a kernel side/userspace change necessary for Flipp to Shh to function.

**Comment by AuroraWright** [comment]
Interesting, the apk does enable the feature on my Pixel 2 but I suppose all Pixels might be predisposed for the feature to be unlocked in the future. The only "unique" thing that Flip to Shhh has (that I can tell) is that it uses the proximity sensor.

**Comment by lzlrd** [comment]
@AuroraWright Flip to Shh relies on gyro too IIRC, but also it's possible that it checks for a sysnode of the Pixel's proximity sensors/gyro which are different on other devices.
